Mathlib/RingTheory/PowerSeries/WeierstrassPreparation.lean

```diff
@@ -331,13 +370,130 @@ theorem eq_of_mul_add_eq_mul_add [IsHausdorff I A] {q q' : A⟦X⟧} {r r' : A[X
   simp_rw [sub_eq_zero] at h
   exact ⟨h.1, h.2.symm⟩
 
+@[simp]
+theorem div_add [IsAdicComplete I A] {f f' : A⟦X⟧} : H.div (f + f') = H.div f + H.div f' := by
```

Comment:
I think `f` and `f'` should be explicit here and in the lemmas below (and the fact that you have to use `a := 0` confirms this).

Reply:
You can even use a `variable` line with `f`, `f'` and `a`